engine.py

```diff
@@ -507,9 +507,13 @@ def save_to_path(self, document, path):
         
             (_, ext) = os.path.splitext(path)
             ext = ext.lower()
-
-            # these extensions are supported by photoshop      
-            if ext ==".bmp":
+          
+            if ext == ".psb":
```

Comment:
I'd break this out into a separate conditional statement that runs first, just to make it clear that it is separate from the remaining extension checks. Also suggest a comment explaining why it is separate.

```python
# comment explaining why this is separate
if ext == ".psb":
    ...
    return

# the remaining extensions follow the same pattern of defining 
# save options that will be supplied to the document's saveAs 
# method.
if "ext" == ".bmp":
    ...
```